src/app/team/team-list-client.tsx

```
@@ -0,0 +1,98 @@
+
+      {/* Content Layer */}
+      <div className="relative z-10 flex-1 pt-20">
+        <TeamNavbar
+          filterMode={filterMode}
+          onFilterChange={setFilterMode}
+          resultCount={filteredMembers.length}
+        />
+
+        <main
+          id="main-content"
+          role="main"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<main>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)